docs/content/commands/npm-publish.md

```diff
@@ -132,6 +132,11 @@ If you want your scoped package to be publicly viewable (and installable)
 set `--access=public`. The only valid values for `access` are `public` and
 `restricted`. Unscoped packages _always_ have an access level of `public`.
 
+Note: Using the `--access` flag on the `npm publish` command will only set
+the package access level on the initial publish of the package. Any subsequent `npm publish`
+commands using the `--access` flag will not have an effect to the access level.
+To make changes to the access level after the initial publish use `npm access`.
+
```

Comment:
this implies that `npm publish` with `--access` on a successive publish should throw, rather than silently fail

Reply:
Let me know what you think if I remove the sentence.

```suggestion
Note: Using the `--access` flag on the `npm publish` command will only set
the package access level on the initial publish of the package.
To make changes to the access level after the initial publish use `npm access`.

```

Reply:
i think either sentence is good; i'm suggesting a broader change to the npm cli's runtime behavior itself.

Reply:
Either would be fine imo. Either mark this suggestion as resolved, or commit it, and I think it's a 👍